claim/models.py

```diff
@@ -1,17 +1,57 @@
 import datetime
 import json
 
+from django.db.models import Q
 from django.db import models
 from django.contrib.auth.models import User
-from django.conf import settings
 from django.db.models import Count
 from django.utils.translation import ugettext as _
 
 
+class Organization(models.Model):
+    id = models.IntegerField(primary_key=True)
+    name = models.CharField(max_length=255)
```

Comment:
I propose to go to DB based map. Probably most correct here will be usage of MongoDB, but to prevent multiple DB's i think we should start migrate all information to exists SQL DB.

Of course we will need simple tool to export/import geojson.

Roadmap in my head:
 - Step by step duplicate all information from geojson in DB (on this tep we will keep information in both places - filesystem and DB)
 - Create import/export tool (Here we can easily sync both sources of truth.)
 - Disable original geojson usage (On this step we will have pure DB based map. And way to export/modify/import.)